src/System.Numerics.Vectors/tests/Matrix3x2Tests.cs

```diff
@@ -1009,7 +1009,7 @@ struct Matrix3x2PlusFloat_2x
 
         // A test to make sure the fields are laid out how we expect
         [Fact]
-        [ActiveIssue(1002)]
+        [ActiveIssue("NullReferenceException")]
```

Comment:
I didn't realize this was blocking.  The changes were made internally earlier today.  Merged and closed PR.